packages/s3-deploy/static-wado.json5.multidist.sample

```diff
@@ -0,0 +1,38 @@
+{
+  staticWadoConfig: {
```

Comment:
Could you give an example with two OHIF deployments pointing to the same DICOMweb group (eg to ugprade to a new version of DICOMweb), and explain how to run those to upload one at a time to the deployments.
That should have:
2 cloud front distributions - one for each OHIF version
2 s3 OHIF buckets, one for each OHIF
1 s3 dicomweb bucket, shared between the two distributions.

Reply:
I just realized this won't actually work as it is - the s3-deploy code assumes that it's creating the specified bucket - so I'll make some changes to support this scenario.